api/db.py

```diff
@@ -65,6 +72,20 @@ async def find_by_id(self, model, obj_id):
         obj = await col.find_one({'_id': ObjectId(obj_id)})
         return model(**obj) if obj else None
 
+    def _operator_translation(self, attributes):
+        translated = attributes.copy()
```

Comment:
Instead of starting with a copy and then iterating through the attributed to edit it, it would seem easier to follow the implementation if it started with an empty dictionary and then arguments would be added to it either as-is if they're not operators or translated if they are.

Reply:
I think this is just another way of doing it.

Reply:
Yes but I mean the way I'm suggesting it has the advantage of defining / constructing the output only once.  It could even be turned into a comprehension I believe, not that it's necessarily always an improvement in Python.